Add collectionview docs - first part (#762)
telerik/maui-docs: controls/collectionview/grouping/expand-collapse.md

---
title: Expand and Collapse Groups
page_title: .NET MAUI CollectionView Documentation - Expand and Collapse Groups
description: Try the Telerik UI for .NET MAUI CollectionView Expand and Collapse group operations.
position: 3
slug: collectionview-expand-collapse-groups
published: False
tags: group, collectionview, expand, collapse, collapsible, dotnet maui, maui
---

# .NET MAUI CollectionView Expand and Collapse Groups

> The CollectionView supports groups expand and collapse operations through the UI by tapping on the group headers. By default, all the groups are expanded.

Reviewer: ```suggestion
The CollectionView supports expand and collapse operations for the groups. To trigger these operations, the user can tap on the group headers. By default, all the groups are expanded.
```
